### ISSUE TITLE
Add Undo Button

### ISSUE THREAD

**Issue by mdsimpson42** [colouring-cities/colouring-core#1414]
Need undo button as if you add data and realise wrong and can't find alternative you can't undo - also for colouring core. But also need to stop people just deflating other people's entries without adding something new

**Comment by polly64** [comment]
@mdsimpson42 @matkoniecz yes think you must have to replace to delete - it is slightly annoying for the user but a good safety valve - do have a chat on this
